src/clj/game/core/pick_counters.clj

```diff
@@ -111,7 +116,7 @@
                                            (when (and card-strs remainder-str)
                                              " from their credit pool"))]
                           (lose state side :credit remainder)
-                          (let [cards (map :card (vals selected-cards))]
+                          (let [cards (not-cost-reduction-payment-cards (map :card (vals selected-cards)))]
```

Comment:
I think I would prefer to inline this function call: `(->> (vals selected-cards) (map :card) (remove #(-> (card-def %) :interactions :pay-credits :cost-reductions)))`. It's not being used anywhere else and the name is almost the same length as the implementation.

Reply:
Thanks. I scoped it into a function because it seemed hard to read but `->>` definitely makes this much easier to understand